labelme/widgets/label_dialog.py

```diff
@@ -103,7 +110,15 @@ def __init__(
         self.resetFlags()
         layout.addItem(self.flagsLayout)
         self.edit.textChanged.connect(self.updateFlags)
+        self.delObjAttrsTextRangeFields
+        self.deleteRangeLayout
+        self.customAttrsIndxs = []
+        self.customAttrsRange = []
+        self.radioButtonsLayout = QtWidgets.QVBoxLayout()
+        layout.addItem(self.radioButtonsLayout)
+        self.resize(400, 300)
```

Comment:
is it supposed to be fixed size? Any auto size?

Reply:
Removed that line! Added parameter in the user config file for this, since I do not like how autosizing is done!